lumispy/tests/components/test_convolved_gaussian_exponential.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019-2025 The LumiSpy developers
+#
+# This file is part of LumiSpy.
+#
+# LumiSpy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the license, or
+# (at your option) any later version.
+#
+# LumiSpy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with LumiSpy. If not, see <https://www.gnu.org/licenses/#GPL>.
+
+import pytest
+
+import numpy as np
+import hyperspy.api as hs
+
+from lumispy.components import ConvGaussExp
+
+sqrt2pi = np.sqrt(2 * np.pi)
+sigma2fwhm = 2 * np.sqrt(2 * np.log(2))
+
+
+def test_function():
+    g = ConvGaussExp(height=3, t0=35, sigma=10, tau=100)
+    assert g.function(-50) == 0.0
+    np.testing.assert_allclose(g.function(35), 1.3874364)
+
+
+class TestConvGaussExp:
```

Comment:
Add a test that check that there is no normalisation issue: for example by making a dataset where the Gaussian convolution is negligible and checking that the fitting parameter are very close for the `ConvGaussExp` and `Exponential` components.